.coverage

```diff
@@ -1 +1 @@
-!coverage.py: This is a private format, don't read it directly!{"lines":{"/vagrant/service/__init__.py":[6,7,8,9,10,13,14,17,19,21,22,23,27,32,33,34,36,37,43],"/vagrant/service/service.py":[11,13,14,15,16,17,18,19,21,34,39,48,57,66,75,86,94,114,118,126,129],"/vagrant/service/models.py":[1,2,3,7,8,9,16,19,20,21,24,25,26,27,29,32,40,47,68,84,71,72,75,76]}}
\ No newline at end of file
+!coverage.py: This is a private format, don't read it directly!{"lines":{"/vagrant/service/__init__.py":[6,7,8,9,10,13,14,17,19,21,22,23,27,32,33,34,36,37,43],"/vagrant/service/service.py":[11,13,14,15,16,17,18,20,21,22,24,37,42,51,60,69,78,89,97,117,123,134,137,105,106,107,108,109,115,119,120,121],"/vagrant/service/models.py":[1,2,3,7,8,9,16,19,20,21,24,25,26,27,28,30,40,47,67,73,84,95,76,77,80,81,42,43,44,45,54,55,56,57,59,65,87,88]}}
```

Comment:
Let's avoid pushing .coverage and .pyc
Possibly you could do it by git rm service/*.pyc and then commit it and push it to this branch. I am not sure whether it works but worth of trying

Reply:
git rm service/*.pyc is not working, same as git rm tests/*.pyc
`fatal: pathspec 'service/*.pyc' did not match any files`